buildSrc/src/main/kotlin/Train.kt

```diff
@@ -90,7 +90,7 @@ fun Project.getKotlinLanguageVersion(): KotlinVersion =
 private fun Project.resolveKotlinLanguageVersion(): KotlinVersion {
     val languageVersion = rootProject.findProperty("kotlin_language_version")
         ?.let { KotlinVersion.fromVersion(it.toString()) }
-        ?: KotlinVersion.KOTLIN_2_0
+        ?: KotlinVersion.KOTLIN_2_1
```

Comment:
What is our strategy? Should we increase API and language versions, or should we keep them as low as possible?

Reply:
I think it would be nice to increase them for a reason

Reply:
I think we can bump language level to `2.1`, but keep API level `2.0`. This should keep Ktor compatibility with Kotlin 2.0, but give us benefits from enabled progressive mode.